core/modules/search/search.install

```diff
@@ -175,6 +206,13 @@ function search_update_1000() {
   $config->set('search_active_modules', update_variable_get('search_active_modules', array('node', 'user')));
   $config->set('search_and_or_limit', update_variable_get('search_and_or_limit', 7));
   $config->set('search_default_module', update_variable_get('search_default_module', 'node'));
+  
+  // Save content types to index.
+  $content_types = array();
+  foreach (node_type_get_types() as $type => $content_type) {
+    $content_types[$type] = $type;
+  }
+  $config->set('content_types', $content_types);
```

Comment:
This needs to be in a new update hook so that it runs for existing Backdrop sites, i.e. `search_update_1002()`.